js/musicutils.js

```diff
@@ -286,6 +286,24 @@ var MODENAMES = [
     [_('custom'), 'custom'],
 ];
 
+var OCTAVESELECTORS = [
+    [-1,-1],
```

Comment:
I don't think we need the entries for -1, 0, and 1 since they are the same as previous, current, and next.